### ISSUE TITLE
New/update info boxes

### ISSUE THREAD

**Issue by ntncmch** [epicentre-msf/gtfcc-ocv-app#17]
- [ ] Add new info box in 3rd position with Number of doses shipped (main label) and % shipped with ICG (sub-label). Use a plane as logo?
- [ ] Modify "Countries" info box: replace WHO regions by N (%) countries with request approved (some countries never had a request approved)

Thanks

**Comment by PaulC91** [comment]
hey @ntncmch I've been meaning to test out the new bslib package for shiny UI which uses the newest version of bootstrap and has a lot of nice features including more flexible column layouts, so I thought this issue was a good incentive to give it a try... https://apps.epicentre-msf.org/public/app_direct/gtfcc 

If you hover over a 'card' element a button will appear on the bottom right of the card to make it full window-size which I think is a really nice feature! Also the collapsible sidebar is cool.

The only issue is the 'capture' package I was using for the map screenshot stopped working with this UI for some reason so I removed the map download option, but I think the option to make the map the full window size should make quite easy for people to resize to their liking and take a screenshot themselves.

I also changed a few colours to match [their website](https://www.gtfcc.org/). 

Let me know your thoughts !
